### ISSUE TITLE
CNN classifier for developed vs not developed images

### ISSUE THREAD

**Issue by JWeb56** [UNCG-CSE/Poststorm_Imagery#197]
**This issue is for training and testing a CNN classifier to determine whether a given image is developed or not, i.e. whether there are manmade structures such as buildings**
This will involve using the Keras/Tensorflow libraries in Python

**First Part**
Select a training set of tagged images of as large a size as possible with approximately equal number of developed and undeveloped images. Convert images to grayscale and remove any extra tag information from the labeled data (i.e. information about washover, terrain type, etc.).

**Second Part**
Potentially compress images even further if necessary (i.e. training the CNN is taking way too long).

**Third Part**
Use Keras/Tensorflow libraries in Python to train a model on the images. Use an appropriate CNN architecture (such as ResNet or GoogLeNet) with its associated convolutional, dropout, pooling, normalization, and fully-connected layers. Hopefully this can be done in a reasonable amount of time given the size of our images. 

**Fourth Part**
Depending on the size of the original data set, test the model using either cross-validation or a designated portion of the data. Hopefully we can confirm that our model will classify images with > 90% accuracy.

**Comment by JWeb56** [comment]
Completed this task
